sale_commission_pricelist/__manifest__.py

```diff
@@ -3,8 +3,8 @@
 
 {
     "name": "Sales commissions by pricelist",
-    "version": "13.0.1.1.0",
-    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "author": "Tecnativa, Odoo Community Association (OCA), Openforce Srls",
```

Comment:
You can't add yourself as author of the module for such simple module migration. Put yourself as contributor.

Reply:
but there is Tecnativa is normal?

Reply:
Of course. We created the module.

Reply:
@pedrobaeza Done